polyply/src/random_walk.py

```diff
@@ -331,23 +365,27 @@ def _random_walk(self, meta_molecule):
             first_node = self.start_node
 
         if "position" not in meta_molecule.nodes[first_node]:
-            constrained = full_fill_geometrical_constraints(self.start, self.molecule.nodes[first_node])
-            if not self._is_overlap(self.start, first_node) and constrained:
-                self.nonbond_matrix.add_positions(self.start, self.mol_idx, first_node, start=True)
+            constrained = fullfill_geometrical_constraints(self.start,
+                                                           self.molecule.nodes[first_node])
```

Comment:
Also don't like this linebreak

Reply:
why?

Reply:
Because it's not too long if you keep it on one line. But like I said, fine to leave it as is.